package.json

```diff
@@ -64,8 +64,16 @@
     "@octokit/rest": "^19.0.5",
     "@octokit/webhooks": "^10.6.2",
     "@octokit/webhooks-types": "^6.9.0",
+    "@opentelemetry/exporter-metrics-otlp-grpc": "^0.40.0",
+    "@opentelemetry/exporter-trace-otlp-grpc": "^0.40.0",
+    "@opentelemetry/id-generator-aws-xray": "^1.1.2",
+    "@opentelemetry/instrumentation-aws-sdk": "^0.34.3",
+    "@opentelemetry/propagator-aws-xray": "^1.2.1",
+    "@opentelemetry/sdk-trace-base": "^1.14.0",
+    "@overbit/opentelemetry-nestjs": "^4.1.0",
```

Comment:
Do we need something that comes directly from `@overbit` in npm? If it's custom, should we not publish it as `@amplication` ?

Reply:
I'm maintaining it outside amplication but we can republish it if you want.